### PR TITLE
api to run configured profiles

### PR DESCRIPTION
**Pull request by badhrinathpa**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,7 +37,7 @@ docker-build:
 			. \
 			|| exit 1; \
 	done
-	sudo rm -rf vendor
+	rm -rf vendor
```
Comment (thakurajayL) [inline Makefile:40]: Wondering why we had added sudo. and do you see any issues with sudo ?

File: profile/httprouter/handler.go [hunk 2]
```diff
@@ -61,6 +63,52 @@ func HTTPAddNewCallsProfile(c *gin.Context) {
 	}
 }
 
+func HTTPExecuteConfigProfile(c *gin.Context) {
+
+	logger.HttpLog.Infoln("ExecuteConfigProfile API called")
+	//var prof profCtx.Profile
```
Comment (thakurajayL) [inline profile/httprouter/handler.go:69]: Do you want to get rid of commented code ?

File: profile/httprouter/handler.go [hunk 3]
```diff
@@ -61,6 +63,52 @@ func HTTPAddNewCallsProfile(c *gin.Context) {
...
+		if config.Configuration.ExecInParallel == false {
+			profileWaitGrp.Wait()
+		}
+	}
+
+	if config.Configuration.ExecInParallel == true {
+		profileWaitGrp.Wait()
+	}
+
+	c.JSON(http.StatusOK, gin.H{})
```
Comment (thakurajayL) [inline profile/httprouter/handler.go:110]: I realized yesterday that if profile takes lot of time then GUI waits for long time. Perhaps you respond to GUI immediately and then continue execution in background. If new API request comes while previous execution is still in progress then response with 200 again.

**Review by thakurajayL — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (badhrinathpa) [reply Makefile:40]: not sure when we added it. but when we try to run on the laptop with sudo it asks for password. so thought better to remove it. not needed in any case.

File: profile/httprouter/handler.go (hunk 3 above)
Comment (badhrinathpa) [reply profile/httprouter/handler.go:110]: we should send the result of the execution in the response. had done it in the gui branch. will merge to this. its better than sending empty response.

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**